src/vm/method.hpp

```diff
@@ -2796,16 +2821,24 @@ class NDirectMethodDesc : public MethodDesc
         return ndirect.m_pWriteableData;
     }
 
+    PTR_NDirectImportThunkGlue GetImportThunkGlue()
```

Comment:
This should be merged with GetNDirectImportThunkGlue - it does the same thing. It is fine to change the return type to `PTR_NDirectImportThunkGlue`.